ci/cron/src/Main.hs

```diff
@@ -168,6 +168,14 @@ build_docs_folder path versions latest = do
         copy from to = do
             shell_ $ "cp -r " <> from <> " " <> to
         build version path = do
+            case version of
+              "0.13.43" -> do
+                  shell_ "git checkout 0c4f9d7f92c4f2f7e2a75a0d85db02e20cbb497b"
```

Comment:
Maybe it would make more sense to just do a `git cherry-pick` of https://github.com/digital-asset/daml/commit/0c4f9d7f92c4f2f7e2a75a0d85db02e20cbb497b? That should be faster (not that this matters much given that we only build it once) but more importantly it should work regardless of what Bazel does wtr to caching. The obvious downside is that the docs are not build from the actual release commit but for this particular change this seems acceptable especially given that all the artifacts have hashes afaik so if the URL change resulted in unrelated changes the build would fail.